### ISSUE TITLE
Screenreaders can't type text

### ISSUE THREAD

**Issue by brandynbennett** [cibernox/ember-power-select-typeahead#67]
Because the `ember-basic-dropdown` trigger element has an aria-role of `button` a screenreader is unable to enter the typeahead area to do any typing. 

I think it would make sense to make a `triggerRole` property in `ember-power-select` that would be passed through to the `role` attribute on the `ember-basic-dropdown` trigger.

The `<input>` on the `ember-power-select-typeahead/trigger` should also have a role of `combobox`

![power-select-typeahead-a11y-issue](https://user-images.githubusercontent.com/4146629/32299481-5f38eac8-bf1b-11e7-9043-04d5184d0854.gif)

**Comment by acorncom** [comment]
Looks like this issue can be closed
